### DISCUSSION TITLE
What does the function(mask_image)  do?

### DISCUSSION THREAD

**Discussion by glorioustory** [royorel/Lifespan_Age_Transformation_Synthesis#29] [General]
def mask_image(self, img, parsings):
        labels_to_mask = [0,14,15,16,18]
        for idx in labels_to_mask:
            img[parsings == idx] = 128
        return img
We look forward to your answer，thanks.

**Comment by royorel** [comment]
@glorioustory 

This is the function that creates the grey background of the images. It removes the background and keeps just the face.

**Reply by glorioustory** [reply]
thanks for answer, your work is great!  the [0,14,15,16,18] mean the background index which generated by semantic segmentation model from DeepLabV3 ?

**Reply by royorel** [reply]
Yes, these are the indices for background, hats, earrings, necklaces and clothing items respectively.
